README.md

```diff
@@ -163,280 +173,41 @@ const order: MarketOrderSpec = {
-  .then((res) => console.log(`Response from provider: ${exe1.provider.name} (ip: ${exe1.getIp()})`, res.stdout));
-```
-
-[Check the full example](./examples/basic/vpn.ts)
-
-### Events
-
-You can listen to various events that are emitted by the SDK. This is useful when you want to react to certain
-conditions, like calculating the total cost of all invoices received.
-
-```ts
-glm.payment.events.on("invoiceAccepted", (invoice) => {
-  console.log("Invoice '%s' accepted for %s GLM", invoice.id, invoice.amount);
-});
-```
-
-[Check the full example](./examples/basic/events.ts)
-
-### Custom filters
-
-You can define custom filters to select the providers that you want to work with. This is useful when you want to
-blacklist or whitelist certain providers.
-
-```ts
-const myFilter: ProposalFilter = (proposal) => proposal.provider.name !== "bad-provider";
-
-const order: MarketOrderSpec = {
-  market: {
-    proposalFilter: myFilter,
-    // other options
-  },
-};
-```
-
-[Check the full example](./examples/advanced/proposal-filter.ts)
-
-We have also prepared a set of predefined filters for common use-cases. [Check out the example with predefined filters here](./examples/advanced/proposal-predefined-filter.ts)
-
-### Custom ranking of proposals
-
-You can define a method that will select which proposal should be chosen first. This is useful when you want to
-prioritize certain providers over others.
-
-```ts
-const scores = {
-  "very-good-provider": 10,
-  "good-provider": 5,
-  "bad-provider": -10,
-};
-
-const bestProviderSelector = (proposals: OfferProposal[]) => {
-  return proposals.sort((a, b) => (scores[b.provider.name] || 0) - (scores[a.provider.name] || 0))[0];
-};
-
-const order: MarketOrderSpec = {
-  market: {
-    proposalSelector: bestProviderSelector,
-    // other options
-  },
-};
-```
-
-[Check the full example](./examples/advanced/proposal-selector.ts)
-
-### Uploading local images to the provider
-
-You can avoid using the registry and upload a GVMI image directly to the provider. This is useful when you want to
-quickly prototype your image without having to update the registry with every change.
-
-```ts
-const order: MarketOrderSpec = {
-  demand: {
-    workload: {
-      imageUrl: "file:///path/to/your/image.gvmi",
-    },
-    // other options
-  },
-};
-```
-
-[Check the full example](./examples/advanced//local-image/)
-
-### Setup and teardown methods
+### Read more
 
-You can define a setup method that will be executed the first time a provider is rented and a teardown method
-that will be executed before the rental is done. This is useful when you want to avoid doing the same work
-multiple times when running multiple tasks on the same provider.
+- Read about [other available usage patterns](./docs/USAGE.md) to learn more on how you can leverage the SDK.
+- Check the [`examples` directory](./examples) for working examples showcasing various features of the SDK. All examples
+  are continuously E2E tested.
```

Comment:
This is basically a repeat of what's directly below, IMO we should just keep the ## Examples section and refator this into a ## Usage (or ## Features) section

```md
## Features
Read about [other available usage patterns](./docs/USAGE.md) to learn more on how you can leverage the SDK.

## Examples
...
```